_pages/en_US/region-changing.txt

```diff
@@ -72,15 +72,29 @@ To use the [magnet](https://wikipedia.org/wiki/Magnet_URI_scheme) links on this
 
 {% include_relative include/ctrtransfer-ticket-copy.txt %}
 
-#### Section VI - Region settings
+#### Section VI - Deleting system save data
 
-1. Launch the System Settings
-1. Go to "Other Settings", "Profile", then "Region Settings"
-1. Select a country from the region you switched to
-1. If prompted, you do not need to set a state
-
-#### Section VII - Remove CTRTransfer image
-
-{% include_relative include/ctrtransfer-cleanup.txt %}
+1. Power off your device
+1. Press and hold (Start), and while holding (Start), power on your device. This will launch GodMode9
+1. Navigate to `[0:] SDCARD` -> `gm9`
+1. Press (X) on the CTRTransfer image `.bin` to delete it
+1. Press (A) to confirm
+1. Press (B) a few times to go back to the main menu
+1. Navigate to `[1:] SYSNAND CTRNAND` -> `data` -> `<ID0>` -> `sysdata`
+  + The `<ID0>` is a 32 characters long
+1. Use the D-Pad to highlight `00010017`
+1. Press (Right Shoulder) + (A) to bring up the folder options
+1. Select "Copy to 0:/gm9/out"
+1. Press (A) to continue
+1. While still highlighting `00010017`, press (X) to delete it
+1. Press (A) to confirm
+1. Press (A) to unlock SysNAND (lvl2) writing, then input the key combo given
+1. Once the file has been deleted, press (A) to continue
+1. Press (A) to relock write permissions if prompted
+1. Press (Start) to reboot your device
+1. Complete the 3DS initial setup as normal
```

Comment:
Could use some more explaining for end user, otherwise seems to be ok overall

Reply:
initial setup

guess it really doesn't matter either way, just thought it would be nice to add at least a small note that it boots into first time setup and this is normal and you didn't lose data
small reassurance goes a long way they say